readme.txt

```diff
@@ -313,8 +313,10 @@ The plugin is made with love by [Modern Tribe Inc](http://m.tri.be/2s).
 * New - Made the "events" and "event" slugs translatable by WPML and other multilingual plugins [95026]
 * New - Introduced the `tribe_events_query_force_local_tz` filter to allow for forcing non-UTC event start and end times in Tribe__Events__Query [92948]
 * Fix - Allow The Events Calendar REST API to be disabled using the `tribe_events_rest_api_enabled` filter [97209]
-* Tweak - Added some more context to the labeling of the "Number of events per page" option (thanks to Todd H. for highlighting this label) [73659]
 * Fix - Remove the errant `div.tribe-events-single-section` on the single event view when there is no venue [97615]
+* Tweak - Improved the alignment of the event cost on the single-event view (props to @canberraclimbersassociation for reporting this bug) [97208] 
+* Tweak - Added some more context to the labeling of the "Number of events per page" option (thanks to Todd H. for highlighting this label) [73659] 
```

Comment:
Duplicated entry of the Changelog.

Reply:
I was rearranging things in the readme, these aren't "new" with this PR just rearranged.

Reply:
Right, but this line appears twice.